### PR TITLE
Add block rep instructions

### PR DESCRIPTION
**Pull request by vincenzocoia**
You can ignore the initial commits on the stat blog post. That's on another branch now.

### CODE REVIEW

File: resources_pages/block_rep.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Block Representative Guidelines
+
+To assist in this, every block, a student volunteers to represent their peers, acting as a liaison between students and MDS leadership (we/our/ours). This allows us to see how the block is going, allowing us to make changes if need be.
```
Comment (mgelbart) [inline resources_pages/block_rep.md]: What is "this"? I'm a bit lost.

File: resources_pages/block_rep.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#### 2. Collect feedback throughout the block
+
+The main responsibility is to collect feedback from your classmates. How you do this is up to you. Some block reps use online surveys, or even invite students to meet them in person during a dedicated time.
+
+Each block, we will be dedicating a time during lecture for students to discuss the block as a class. This is a great avenue for the block rep to record some key insights. 
+
+#### 3. Attend our team meeting
+
+We'll ask you to attend our team meeting, usually about 2/3 or 3/4 through the block. Typically the block rep part of the meeting takes about 20 minutes, right at the start, but it’s flexible. We’ll continue the meeting without the block rep after they're done.
```
Comment (mgelbart) [inline resources_pages/block_rep.md]: they-->you, for consistency? 
Maybe add that we'll continue to reflect on / discuss what we heard?

File: resources_pages/block_rep.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#### 4. Communicate back to the cohort
+
+Often, after the team meeting, addressing student comments/concerns will involve us getting in contact with the cohort (such as through GitHub Issues). Other times (_e.g._ to preserve anonymity), we will ask the block rep to contact students. You can probably do this via Slack.
```
Comment (mgelbart) [inline resources_pages/block_rep.md]: I don't understand the anonymity issue here?

File: resources_pages/block_rep.md (hunk 2 above)
Comment (mgelbart) [reply resources_pages/block_rep.md]: Maybe we can also add here (or its own section) that, optionally, they can create a short informal report (usually Markdown) which we'll archive on our end.

**Review by mgelbart — CHANGES_REQUESTED** [review]
Thanks for putting this together @vincenzocoia ! A few nitpicks but looks good overall.

File: resources_pages/block_rep.md (hunk 1 above)
Comment (vincenzocoia) [reply resources_pages/block_rep.md]: Oops. I had deleted a previous sentence!

File: resources_pages/block_rep.md (hunk 2 above)
Comment (vincenzocoia) [reply resources_pages/block_rep.md]: Adding to Section 2

File: resources_pages/block_rep.md (hunk 3 above)
Comment (vincenzocoia) [reply resources_pages/block_rep.md]: I was trying too hard. I'll simplify.

**Review by mgelbart — APPROVED** [review]

**Pull request merged**